<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchDialog</class>
 <widget class="QDialog" name="SearchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>795</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabs">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Search</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QComboBox" name="match_kind">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <item>
          <property name="text">
           <string>Wildcard</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Regex</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="search_box">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Final selection</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" rowspan="2">
        <widget class="QListWidget" name="matches">
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="add_btn">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="resource.qrc">
           <normaloff>:/left.png</normaloff>:/left.png</iconset>
         </property>
        </widget>
       </item>
       <item row="2" column="2" rowspan="2">
        <widget class="MinimalListWidget" name="selection"/>
       </item>
       <item row="3" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="0" colspan="3">
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0,0,0">
         <item>
          <widget class="QPushButton" name="cancel_btn">
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="add_window_btn">
           <property name="text">
            <string>Add window</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="apply_btn">
           <property name="text">
            <string>Apply</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="status">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Pattern based window</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3" rowstretch="0,0,0,0,1,0,0,0" columnstretch="0,0,0,1,0">
       <item row="3" column="1">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Filter value</string>
         </property>
        </widget>
       </item>
       <item row="7" column="4">
        <widget class="QPushButton" name="apply_pattern_btn">
         <property name="text">
          <string>Apply</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QDoubleSpinBox" name="filter_value">
         <property name="decimals">
          <number>6</number>
         </property>
         <property name="minimum">
          <double>-9999999999999999635896294965248.000000000000000</double>
         </property>
         <property name="maximum">
          <double>999999999999999983222784.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Pattern</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Filter type</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>254</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="7" column="1">
        <widget class="QPushButton" name="cancel_pattern_btn">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item row="7" column="3">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>282</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="2">
        <widget class="QLineEdit" name="pattern">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QPushButton" name="define_ranges_btn">
         <property name="text">
          <string>Define ranges</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QComboBox" name="pattern_match_type">
         <item>
          <property name="text">
           <string>Wildcard</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Regex</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QComboBox" name="filter_type">
         <item>
          <property name="text">
           <string>Unspecified</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Contains</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Do not contain</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Constant</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Match type</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QCheckBox" name="raw">
         <property name="text">
          <string>Asses the raw channel values</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MinimalListWidget</class>
   <extends>QListWidget</extends>
   <header>asammdf/gui/widgets/list.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>search_box</tabstop>
  <tabstop>matches</tabstop>
  <tabstop>add_btn</tabstop>
  <tabstop>selection</tabstop>
  <tabstop>add_window_btn</tabstop>
  <tabstop>apply_btn</tabstop>
  <tabstop>cancel_btn</tabstop>
  <tabstop>match_kind</tabstop>
 </tabstops>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
